app/views/guides/_form.html.erb

```diff
@@ -72,8 +72,29 @@
       <div class='form-group'>
         <%= f.submit "Save Draft", class: 'btn btn-success', name: :save_draft %>
         <%= link_to "Preview", document_preview_url(guide), class: 'btn btn-default' %>
-        <%= f.submit "Publish", class: 'btn btn-success', name: :publish %>
+        <% if guide.latest_edition.present? && guide.latest_edition.approved? %>
+          <%= f.submit "Publish", class: 'btn btn-success', name: :publish %>
+        <% end %>
       </div>
     </fieldset>
   <% end %>
+
+  <% if guide.latest_edition.present? %>
+    <% if guide.latest_edition.persisted? && !guide.latest_edition.review_requested? %>
+      <%= form_for :review_request, url: edition_review_requests_path(edition_id: guide.latest_edition.id) do |f| %>
+        <%= f.submit "Send for review", class: 'btn btn-success' %>
+      <% end %>
+    <% end %>
+
+    <% if guide.latest_edition.persisted? && guide.latest_edition.review_requested? %>
```

Comment:
Same here, i think we should check if there is any review request created since the last publication instead of checking the latest edition only.